src/get-started/light-client.md

```diff
@@ -316,34 +439,29 @@ if (typeof wallet === 'undefined') {
 }
 ```
 
-Basically, the proof from the Timestamping Demo comprises the same validation
+Basically, the proof from the Timestamping Service comprises the same validation
 levels as described above. Specifically, to obtain confirmation for the
 timestamp data, the proof validates the block, the root table and the table of
 timestamps. The data on the timestamp can then be extracted from the validated
 timestamps table.
 
-Meanwhile, the proof we presently investigate contains another level that refers
-to the validation of transactions in respect of each wallet.  
+Meanwhile, the proof we presently investigate contains another level. This level
+refers to the validation of transactions in respect of each wallet (wallet
```

Comment:
Is "in respect of..." a real phrase? 🙃 Maybe, something like "concerning a specific wallet" would be better?

Reply:
actually, it is. and it is absolutely suitable for the case but I do not mind changing the text as you suggest